packages/common/src/styles/theme.ts

```diff
@@ -45,7 +46,7 @@ export type ThemeOptions = Modify<
   }
 >;
 
-export type Theme = Modify<
+export type AppTheme = Modify<
```

Comment:
I'm getting issues here.

If I export this from the `styles/index.ts` like:

![image](https://user-images.githubusercontent.com/35858975/128423007-7aadf8dd-91a8-493d-bf82-4a8989073a60.png)

and then try to import it from elsewhere like:
![image](https://user-images.githubusercontent.com/35858975/128423056-853ea57c-6af6-4eed-a11c-a7bfd3823fc9.png)


I get no errors and the type is resolved in the IDE fine.

However when I try to compile I get

![image](https://user-images.githubusercontent.com/35858975/128422973-a0e493a6-f726-4eb4-9885-56a6b528b136.png)

Possibly something to do with import order or something 🤔